frontend/src/System/Logs/Files/LogFiles.js

```diff
@@ -76,13 +77,13 @@ class LogFiles extends Component {
         <PageContentBody>
           <Alert>
             <div>
-              Log files are located in: {location}
+              {translate('LogFilesLocation')}: {location}
```

Comment:
`location` should be part of the translation so languages that prefer a different syntax can translate it properly.